src/main/scala/org/scalastyle/Output.scala

```diff
@@ -46,9 +46,7 @@ trait Output[T <: FileSpec] {
   def output(messages: java.util.List[Message[T]]): OutputResult = privateOutput(messages.asScala)
 
   private[this] def privateOutput(messages: Iterable[Message[T]]): OutputResult = {
-    messages.foreach { m =>
-      eachMessage(m); message(m)
-    }
+    messages.foreach { m => eachMessage(m); message(m) }
```

Comment:
Since we are already on `2.13` what about:

```scala
messages.tap(eachMessage).foreach(message)
```

Reply:
sure, I didn't really want to touch this, but scalafmt had a different opinion :)